StyleGuide.md

```diff
@@ -0,0 +1,37 @@
+# RetroConnect Team Style Guide
+---
+
+## Areas of Focus:
+---
+* **External Documentation**
+* **Class Documentation**
+* **Interal Documentation & Naming**
+* **Indentation**
+* **Miscellaneous**
+---
+#### External Documentation:
+* At the top of each file created, the name of the team member working on this file and a brief description of the purpose of the file should be added.
+#### Class Documentation:
+* Each support class is to be preceded with a block comment that includes:
+   * Class name
+   * Names of external packages the class depends on
+   * Parameter names and brief description
+   * Brief note on the classes purpose
+
+#### Internal Documentation & Naming:
+* It is encouraged to name variables and methods descriptively, but if variable names become scarce inline comments are necesary for future readability.
+* *Complex* pieces of code are encouraged to be commented heavily.
+
+
+#### Indentation:
+
+* RetroConnect is a **Tab** operated team with **tab stop = 4**. The use of spaces by a RetroConnect employee will result in immediate termination.
+
+
+#### Miscellaneous:
+* The following guidelanes are reccomended to follow when writing any code:
+  * Write programs with modularity; that is, create classes when appropriate, write methods that accomplish limited and well-defined tasks
+  * **Limited** or very monitored use of Global Variables
+  * Indent bodies of methods, loops and IF statements with consistent style
```

Comment:
What style are we using for loops and if statements?

```cpp
if (condition) {
        // Statements
}

// or

if (condition)
{
        // Statements
}
```
Personally, I am partial to the first style.

Reply:
First style definitely. Think of how many enter key presses could be saved!